velox/functions/prestosql/ArrayFunctions.h

```diff
@@ -513,6 +513,26 @@ struct ArrayConcatFunction {
       out.add_items(array.value());
     }
   }
+
+  FOLLY_ALWAYS_INLINE void call(
```

Comment:
no  need to inline since the function is expensive and no clear value for inlining it